sky/packages/sky_engine/lib/_embedder.yaml

```diff
@@ -12,3 +12,30 @@ embedded_libs:
   "dart:ui": "../sdk_ext/dart_ui.dart"
   "dart:_internal": "../dart_sdk/_empty.dart"
   "dart:nativewrappers": "../dart_sdk/_empty.dart"
+
+analyzer:
+  language:
+    enableSuperMixins: true
+  errors:
+    # We allow overriding fields (if they use super, ideally...).
+    strong_mode_invalid_field_override: ignore
+    # We allow type narrowing.
+    strong_mode_invalid_method_override: ignore
+    todo: ignore
+
+linter:
+  rules:
+    - camel_case_types
+    # Sometimes we have no choice (e.g. when matching other platforms).
+    # - constant_identifier_names
+    - empty_constructor_bodies
+    # Disabled until regexp fix is pulled in (https://github.com/flutter/flutter/pull/1996).
+    # - library_names
+    - library_prefixes
+    - non_constant_identifier_names
+    # Too many false-positives; code review should catch real instances.
+    # - one_member_abstracts
```

Comment:
Hm, once we have per-line ignores, maybe we should use those instead of disabling the lints...